ui/components/component-library/avatar-account/avatar-account.stories.js

```diff
@@ -25,18 +25,18 @@ export default {
   argTypes: {
     size: {
       control: 'select',
-      options: Object.values(AVATAR_ACCOUNT_SIZES),
+      options: Object.values(AvatarAccountSize),
```

Comment:
Yay enums

<img width="583" alt="Screenshot 2023-02-10 at 12 29 47 PM" src="https://user-images.githubusercontent.com/8112138/218191509-392b680e-1657-485c-8e36-416a6514196e.png">

Reply:
Ops... how about using some util func like `getEnumOptions` to extract the values ...  
```
const getEnumOptions = enum =>
 Object.keys(enum).reduce((acc:any[], curr, index, arr) => {
      if (index < arr.length / 2) acc.push(curr);
      return acc;
    }, []);
```

```

Reply:
Thank you! I'm researching to see if there are any best practices with enums and storybook. @brianacnguyen are you aware of any?

Reply:
So this was because I was trying to use the Size enum from `design-system.ts` like this. I've changed it to strings but now we've lose the connection...

```
export enum AvatarAccountSize {
  Xs = Size.XS,
  Sm = Size.SM,
  Md = Size.MD,
  Lg = Size.LG,
  Xl = Size.XL,
}
```